query/graphql/src/main/java/org/codice/ddf/admin/graphql/transform/GraphQLTransformScalar.java

```diff
@@ -17,14 +17,32 @@
 import org.codice.ddf.admin.graphql.GraphQLTypesProviderImpl;
 
 import graphql.Scalars;
+import graphql.schema.Coercing;
 import graphql.schema.GraphQLScalarType;
 
 public class GraphQLTransformScalar {
 
     GraphQLTypesProviderImpl<GraphQLScalarType> scalarTypesProvider;
+    private Coercing encryptPasswordCoercing;
 
     public GraphQLTransformScalar() {
         scalarTypesProvider = new GraphQLTypesProviderImpl<>();
+        encryptPasswordCoercing = new Coercing() {
+            @Override
+            public Object serialize(Object input) {
+                return "***";
```

Comment:
We should change this to a constant and have a discussion about what we want this to be.